README.md

```diff
@@ -1 +1,166 @@
-# i18n
\ No newline at end of file
+# i18n
+
+[![Run Tests](https://github.com/gin-contrib/gin-i18n/actions/workflows/go.yml/badge.svg)](https://github.com/gin-contrib/gin-i18n/actions/workflows/go.yml)
+[![CodeQL](https://github.com/gin-contrib/gin-i18n/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/gin-contrib/gin-i18n/actions/workflows/codeql-analysis.yml)
+[![codecov](https://codecov.io/gh/gin-contrib/i18n/branch/master/graph/badge.svg?token=QNMN3KM28Y)](https://codecov.io/gh/gin-contrib/i18n)
+[![GoDoc](https://godoc.org/github.com/gin-contrib/gin-i18n?status.svg)](https://godoc.org/github.com/gin-contrib/gin-i18n)
+[![Go Report Card](https://goreportcard.com/badge/github.com/gin-contrib/gin-i18n)](https://goreportcard.com/report/github.com/gin-contrib/gin-i18n)
+
+## Usage
+
+Download and install it:
+
+```sh
+go get github.com/gin-contrib/gin-i18n
+```
```

Comment:
`github.com/gin-contrib/i18n`?

Reply:
@thinkerou I updated this. Thank so much for your feedback!